extension-requests/script.js

```diff
@@ -227,7 +227,21 @@ const getExtensionColor = (deadline, createdTime) => {
   return 'orange-text';
 };
 
+const currentUserDetailsPromise = getSelfUser()
```

Comment:
Is this making call to /users/self api?

Reply:
![image](https://github.com/user-attachments/assets/1cc4f9aa-b29f-47e6-81db-83d4f9dbb3a3)
 yes, this is the function that is being used to fetch the user's details before. Haven't changed anything in this, I'm calling the same.

Reply:
Use new endpoint to user profile details

Reply:
Avoid using any self endpoints, they are already in deprecation process

Reply:
@vikasosmium can you tell what is the new endpoint for self user or where can I find it?

Reply:
Updated the endpoint with `/users?profile=true`